src/wp-includes/fonts/class-wp-font-library.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Font Library class.
+ *
+ * This file contains the Font Library class definition.
+ *
+ * @package    WordPress
+ * @subpackage Font Library
+ * @since      6.4.0
+ */
+
+/**
+ * Font Library class.
+ *
+ * @since 6.4.0
+ */
+class WP_Font_Library {
+
+	/**
+	 * Fonts mime types allowed for each file type.
+	 * Each file type can have multiple mime types depending on the PHP version.
+	 * Currently wp_check_filetype_and_ext() only allows one mime type per file extension.
+	 */
+	const PHP_7_TTF_MIME_TYPE = PHP_VERSION_ID >= 70300 ? 'application/font-sfnt' : 'application/x-font-ttf';
```

Comment:
Is there a unit test for this?

Reply:
Yes, they are coming: https://github.com/WordPress/gutenberg/pull/54844
Porting here when that PR is merged.

Reply:
Done :heavy_check_mark:  https://github.com/WordPress/wordpress-develop/pull/5285/commits/04fe719b817b0c57e34442333a9c3744625cd2c4

Reply:
Noting that this implementation has changed: we now have a function used as a callback for the `upload_mimes` filter. And [as suggested here](https://github.com/WordPress/wordpress-develop/pull/5285#discussion_r1342860869), that function does not accept any arguments.

With the function arg removed, the unit tests would simply reimplement the function logic, and so were removed in https://github.com/WordPress/gutenberg/pull/58667